From ed0cf34219e4560d4295f568f3196ee2cd002882 Mon Sep 17 00:00:00 2001
From: Vinaykumar Masane <vinaykumar.masane@freescale.com>
Date: Thu, 10 Dec 2015 16:36:38 +0530
Subject: [PATCH 27/60] SBR-265 Removed sw_xxx fields from boardinfo

SW_DESC SW_OBJID etc. will be part of confd so removed same from boardinfo

Signed-off-by: Vinaykumar Masane <vinaykumar.masane@freescale.com>
---
 commands/boardinfo.c | 94 ++--------------------------------------------------
 1 file changed, 3 insertions(+), 91 deletions(-)

diff --git a/commands/boardinfo.c b/commands/boardinfo.c
index 8f344c0..2ce0a8d 100644
--- a/commands/boardinfo.c
+++ b/commands/boardinfo.c
@@ -42,8 +42,6 @@
 #define SIZE_P_N 12         /* Max  12 Character Board Part Number */
 #define SIZE_PROD_NAME 16   /* Max  16 Character Board Product Name */
 #define SIZE_PROD_SERIES 16 /* Max  16 Character Board Product Series */
-#define SIZE_SW_DESC 128    /* Max 128 Character Board Software Description */
-#define SIZE_SW_OBJID 32    /* Max  32 Character Board Software Object ID */
 
 #define COUNT_MAC        3 /* 3 MAC-IDs one each for LAN, WAN-1, WAN-2 */
 #define TOTAL_MAC_OCTET  6 /* No. of MAC Octets in MAC ID */
@@ -61,8 +59,6 @@ typedef struct c2krv340_board_info {
   unsigned char p_n[16];             /* Board Serial Number */
   unsigned char prod_name[64];       /* Board Product Name */
   unsigned char prod_series[16];     /* Board Product Series */
-  unsigned char sw_desc[128];        /* Board Software Description */
-  unsigned char sw_objid[32];        /* Board Software Object Id */
 }  __attribute__ ((__packed__)) c2krv340;
 
 char *help_string = "Check Usage: boardinfo <Enter>\n";
@@ -390,66 +386,6 @@ static int command_boardinfo_update(int argc, char *argv[])
   boarddata[count - len ] = type;
   boarddata[count - len + 1] = len - 2;
 
-  printf("\nEnter SOFTWARE Description : ");
-  len = 2;
-  count = count + len;
-  while (len < (SIZE_SW_DESC + 2))
-  {
-    c = getc();
-    if (c == KEY_CR)
-      break;
-
-    if ( isprint(c))
-    {
-      c = toupper(c);
-      putchar(c);
-    }
-    else
-    {
-      printf("\n c = %c : 0x%X Input Error! %s", c, c, help_string);
-      return COMMAND_ERROR;
-    }
-    if ( c == KEY_CR )
-    {
-      break;
-    }
-    len++;
-    boarddata[count++] = c;
-  }
-  type = 0xA1;
-  boarddata[count - len ] = type;
-  boarddata[count - len + 1] = len - 2;
-
-  printf("\nEnter SOFTWARE Objectid : ");
-  len = 2;
-  count = count + len;
-  while (len < (SIZE_SW_OBJID + 2))
-  {
-    c = getc();
-    if (c == KEY_CR)
-      break;
-
-    if ( isprint(c))
-    {
-      c = toupper(c);
-      putchar(c);
-    }
-    else
-    {
-      printf("\n c = %c : 0x%X Input Error! %s", c, c, help_string);
-      return COMMAND_ERROR;
-    }
-    if ( c == KEY_CR )
-    {
-      break;
-    }
-    len++;
-    boarddata[count++] = c;
-  }
-  type = 0xA2;
-  boarddata[count - len ] = type;
-  boarddata[count - len + 1] = len - 2;
-
   fd = open(filename, O_RDWR, 0);
   if (fd < 0) {
     printf("could not open %s!\n", filename);
@@ -612,30 +548,6 @@ static int command_boardinfo_show(int argc, char *argv[])
     count++;
   }
 
-  swdesc_length = buf_boarddata[count + 1];
-  count = count + 2;
-  printf("\nSW_DESC  : ");
-  for (i = 0; i < swdesc_length; i++)
-  {
-    if (isprint(buf_boarddata[count]))
-      putchar(buf_boarddata[count]);
-    else
-      putchar(' ');
-    count++;
-  }
-
-  swobjid_length = buf_boarddata[count + 1];
-  count = count + 2;
-  printf("\nSW_OBJID  : ");
-  for (i = 0; i < swobjid_length; i++)
-  {
-    if (isprint(buf_boarddata[count]))
-      putchar(buf_boarddata[count]);
-    else
-      putchar(' ');
-    count++;
-  }
-
   free(board_data);
   printf("\n");
   return COMMAND_SUCCESS;
@@ -666,9 +578,9 @@ static const __maybe_unused char cmd_boardinfo_help[] =
 "Usage: boardinfo [OPTIONS]\n"
 "\n"
 "options:\n"
-" -m  lists boardinfo MAC ID contents from /dev/spi1.board-info\n"
-" -u  update board-info (default updates info to /dev/spi1.board-info)\n"
-" -s  lists boardinfo contents from /dev/spi1.board-info\n"
+" -m  lists boardinfo MAC ID contents from /dev/spi1.boardinfo\n"
+" -u  update boardinfo (default updates info to /dev/spi1.boardinfo)\n"
+" -s  lists boardinfo contents from /dev/spi1.boardinfo\n"
 " e.g.    boardinfo -u\n"
 "Enter PID : <MAX 11 char>\n"
 "Enter VID : <MAX  3 char>\n"
-- 
2.7.4

